main.go

```diff
@@ -127,7 +127,9 @@ func main() {
 	renewDeadline := leaderElectionLeaseDuration / 2
 	retryPeriod := leaderElectionLeaseDuration / 4
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), config.ManagerOptionsWithNamespaces(setupLog, ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.UserAgent = "datadog-operator-manager"
```

Comment:
I'm open to suggestions for the new manager name. Perhaps `datadog-operator` is sufficient?